Daily doesn't build with QGC_DISABLE_PX4_PLUGIN and QGC_DISABLE_PX4_PLUGIN_FACTORY set to ON

## Expected Behavior

QGC should build normally without PX4 support.

## Current Behavior

The following build time error appears:

...\src\Comms\MockLink\MockLink.h(16): fatal error C1083: Can't open included file: 'px4_custom_mode.h': No such file or directory

## Steps to Reproduce:
1. Set-up build normally.
2. Set QGC_DISABLE_PX4_PLUGIN and QGC_DISABLE_PX4_PLUGIN_FACTORY (not sure which one is the actual culprit) to ON in CustomOptions.cmake or elsewhere.
3. Attempt to build

## System Information
- Operating System: Windows 11
- QGC Version: master
- QGC Build: daily

--- a/src/Comms/MockLink/CMakeLists.txt
+++ b/src/Comms/MockLink/CMakeLists.txt
@@ -27,7 +27,7 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
             Qt6::Core
             Qt6::Positioning
             Comms
-            PX4FirmwarePlugin
+            FirmwarePlugin
             MAVLink
     )
 

--- a/src/Comms/MockLink/MockLink.h
+++ b/src/Comms/MockLink/MockLink.h
@@ -9,11 +9,11 @@
 
 #pragma once
 
+#include "PX4/px4_custom_mode.h"
 #include "LinkInterface.h"
 #include "MAVLinkLib.h"
 #include "MockConfiguration.h"
 #include "MockLinkMissionItemHandler.h"
-#include "px4_custom_mode.h"
 
 #include <QtCore/QElapsedTimer>
 #include <QtCore/QLoggingCategory>

